core/src/com/unciv/logic/civilization/WondersInfo.kt

```diff
@@ -0,0 +1,191 @@
+package com.unciv.logic.civilization
+
+import com.badlogic.gdx.graphics.Color
+import com.unciv.UncivGame
+import com.unciv.logic.city.CityInfo
+import com.unciv.logic.map.TileInfo
+import com.unciv.models.ruleset.Building
+import com.unciv.models.ruleset.Era
+import com.unciv.models.ruleset.QuestName
+import com.unciv.models.ruleset.unique.UniqueType
+import com.unciv.models.translations.tr
+import com.unciv.ui.civilopedia.CivilopediaCategories
+
+class WondersInfo {
```

Comment:
`WondersInfo` vs `WonderInfo` seems a bit confusing so a name change might be helpful.